packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -29,32 +31,35 @@ class FileSearchField extends StatefulWidget {
 class _FileSearchFieldState extends State<FileSearchField>
     with SearchFieldMixin, AutoDisposeMixin {
   AutoCompleteController _autoCompleteController;
-  final Map<String, ScriptRef> _scriptsCache = {};
+
+  final _scriptsCache = <String, ScriptRef>{};
 
   final fileSearchFieldKey = GlobalKey(debugLabel: 'fileSearchFieldKey');
 
   @override
   void initState() {
     super.initState();
 
-    _autoCompleteController = AutoCompleteController();
-    _autoCompleteController.currentDefaultIndex = 0;
+    _autoCompleteController = AutoCompleteController()..currentDefaultIndex = 0;
 
-    addAutoDisposeListener(
-        _autoCompleteController.selectTheSearchNotifier, _handleSearch);
     addAutoDisposeListener(
         _autoCompleteController.searchNotifier, _handleSearch);
+    addAutoDisposeListener(_autoCompleteController.searchAutoCompleteNotifier,
+        _handleAutoCompleteOverlay);
 
-    _autoCompleteController.selectTheSearch = true;
+    SchedulerBinding.instance.addPostFrameCallback((_) => _handleSearch());
```

Comment:
can you add a comment explaining why this needs to be in a post frame callback?

Reply:
Done, thanks!